cmd/tink-worker/worker/container_manager.go

```diff
@@ -0,0 +1,143 @@
+package worker
+
+import (
+	"context"
+	"path"
+	"path/filepath"
+
+	"github.com/docker/docker/api/types"
+	"github.com/docker/docker/api/types/container"
+	"github.com/docker/docker/client"
+	"github.com/packethost/pkg/log"
+	"github.com/pkg/errors"
+	pb "github.com/tinkerbell/tink/protos/workflow"
+)
+
+const (
+	ErrFailedToWait   = "failed to wait for completion of action"
+	ErrFailedToRunCmd = "failed to run on-timeout command"
+
+	InfoWaitFinished = "wait finished for failed or timeout container"
```

Comment:
I don't think these need to/should be public right? They're only used within this package.

Reply:
I can make them private